content/chapter-04/NetworkManagementLab.md

```diff
@@ -1,11 +1,61 @@
 # Lab: Cloud Network Management
 
-## Objectives
-
 * Get familiar with Cloud Provider access control management
-* Review options available for VPN connectivity on the portal
+* You will create three small instances on this lab:
+  * **In-Access**: Accessible from outside your network on port 80
+  * **Middle-Access** Accessible only from **In-Access** and **Only-internal** on port 80
+  * **Only-internal** Accessible from **In-Access** and **Middle-Access**, but without access to the outside world
+* Login to Cloud Provider portal
+* Create a security group to allow access to SSH port 22 and the outside world. Let's call this security group `chapter4`
+* Create a security group to allow access to the outside world. Let's call this security group `chapter4-to-outside`
+* Create **In-Access** instance:
+  * Create a small instance to launch the stress test tool, e.g., AWS EC2 t2.micro, AMI: Ubuntu Server 16.04 LTS (HVM), SSD Volume Type - ami-66506c1c
+  * Provision this instance with 8GB of disk space
+  * Associate it to the `chapter4` and `chapter4-to-outside` security groups
+  * Open an ssh session into the instance and execute the following commands to install and configure [nginx](https://nginx.org/en/)
+    * `sudo apt-get update`
+    * `sudo apt-get install nginx` [Reference](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/)
+    * Check the installation was successful buy using accessing the `nginx` using localhost: 
+      * `curl http://localhost`. You should expect an html document stating the installation was successful.
+    * Change the default index html on nginx to show you the name of the server it came from:
+      * `sudo vi /var/www/html/index.nginx-debian.html`
+      * Edit the line `Welcome to nginx!` to `Welcome to nginx from In-Access`
+      * Check the effect of your changes using `curl http://localhost`
+  * Close the ssh session
+* Repeat the above steps to create the other two instances needed (**Middle-Access** and **Only-intenal**)
+
+
+* Try to access any of the instances public ip or dns from your computer:
+  * `curl http://<instance public ip or dns>/ -v -m 10`
```

Comment:
< and > not playing nicely with markdown being converted to html
also do we need to educate them about dns on aws and azure or public ip address is enough for our purposes?

Reply:
Not really.